app/internal/hebrew_date_view.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+
+from hebrew_numbers import int_to_gematria
+from pyluach import dates, hebrewcal
+
+
+EN_TO_HEB_MONTH_NAMES = {
+    'Tishrei': 'תשרי',
+    'Cheshvan': 'חשוון',
+    'Kislev': 'כסלו',
+    'Teves': 'טבת',
+    'Shvat': 'שבט',
+    'Adar': 'אדר',
+    'Adar Rishon': "אדר(א')",
+    'Adar Sheni': "אדר(ב')",
+    'Nissan': 'ניסן',
+    'Iyar': 'אייר',
+    'Sivan': 'סיוון',
+    'Tamuz': 'תמוז',
+    'Av': 'אב',
+    'Elul': 'אלול',
+}
+
+
+def get_month_name_by_num(date: datetime) -> str:
+    """Returns the Hebrew name date for the specific day.
+
+        Args:
+            date: The requested date.
+
+        Returns:
+            A Hebrew name date.
+        """
+
+    for month in hebrewcal.Year(date.year).itermonths():
+        if date.month == month.month:
+            print(month.name)
+            return EN_TO_HEB_MONTH_NAMES[month.name]
```

Comment:
You don't need the dictionary, just use gettext and let the translators do the work :)

Reply:
it is horrible to use the dictionary?
i return string that combine english and hebrew and the gettext not fit to this(I also spoke with @Gonzom about it)
and also from the pyluach the result in leap year for adar is "adar rishon" and "adar sheni"
and i dont want to translate it like that and that is why i put the value אדר(א') and "אדר(ב')".
Can i keep the like that? :)

Reply:
You can change the dictionary into something like that:

```
EN_TO_HEB_MONTH_NAMES = {
    'Tishrei': gettext('Tishrei'),
    'Cheshvan': gettext('Cheshvan'),
    ...
}
```

And add the Hebrew translations into the translation files.

Reply:
and what to do in the adar case?gettext('Adar a')?

Reply:
No one will understand what "Adar a" means ^^"

You can use either Adar Aleph/Adar Bet, Adar Rishon/Adar Sheni or Adar I/Adar II

Reply:
i edit it now and will upload it in half hour